internal/indexer/moniker.go

```diff
@@ -136,17 +137,20 @@ func (i *Indexer) ensureImportMoniker(identifier string, packageInformationID ui
 // monikerPackage returns the package prefix used to construct a unique moniker for the given object.
 // A full moniker has the form `{package prefix}:{identifier suffix}`.
 func monikerPackage(obj types.Object) string {
+	var pkgName string
 	if v, ok := obj.(*types.PkgName); ok {
-		return strings.Trim(v.Name(), `"`)
+		pkgName = strings.Trim(v.Name(), `"`)
+	} else {
+		pkgName = obj.Pkg().Path()
 	}
 
-	return obj.Pkg().Path()
+	return gomod.NormalizeMonikerPackage(pkgName)
 }
 
-// monikerIdentifier returns the identifier suffix used to construct a unique moniker for the given object.
+// getMonikerIdentifier returns the identifier suffix used to construct a unique moniker for the given object.
```

Comment:
Why change this (but not `monikerPackage`)?

Reply:
It's not super idiomatic in Go to have `getX`, `setX` methods. My biggest concern is consistency, though.